services/callback/src/test/java/app/coronawarn/server/services/callback/controller/ApiExceptionHandlerTest.java

```diff
@@ -34,8 +35,6 @@ public void setup() {
 
   @Test
   void testUnexpectedExceptionsTriggersStatusCode500() throws Exception {
-    doThrow(RuntimeException.class).when(serviceMock).save(any());
```

Comment:
I think we should keep this logic. Right now the InternalServerError is thrown because config is null not because the mock throws an error

Reply:
Maybe we can also add a test to see what is the expected outcome if SouceSystem is not set?